### PR TITLE
Makes the script work in windows using a powershell script.

### PR DESCRIPTION
**Pull request by carlosen0153**
Hi Maxim, 
I have no proper experience writing vim-plugings neither powershell scripts but I made it work in my computer equipped with Windows 8.1 Pro and works pretty well for me. 
Please feel free to change/clean anything you consider necesary.

### CODE REVIEW

File: plugin/mdip.vim [hunk 1]
```diff
@@ -67,6 +87,9 @@ function! SaveNewFile(imgdir, tmpfile)
 endfunction
 
 function! RandomName()
+  if has('win16') || has('win95') || has('win32') || has('win64')
+    return
```
Comment (ferrine) [inline plugin/mdip.vim:91]: Why no return here?

**Review by ferrine — COMMENTED** [review]
Oh, sorry for late response, seems like my message is still pending after I reviewed changes

File: plugin/mdip.vim (hunk 1 above)
Comment (carlosen0153) [reply plugin/mdip.vim:91]: If I do not return, when I use the script in windows, I get the following error:
Error detected while processing function mdip#MarkdownClipboard[3]..RandomName:
line 4:
E484 Can't open file C:\Users\Carlos\AppData\Local\Temp\VIo95CA.tmp
This is the reason why the random file name is create in the powershell script.

Comment (ghost) [reply plugin/mdip.vim:91]: similar problems with $RANDOM while I'm using fish shell.
$RANDOM return nothings on fish, so I had to do this on top.
set shell=bash
I already have this in my vimrc, but seems like it doesn't apply to system command inside 
vim plugin..?

By the way, thanks for this great plugin!

Comment (ferrine) [reply plugin/mdip.vim:91]: I do not use Windows at all so i can't even check this works. In here I need more help from the community side. :)

Comment (ferrine) [reply plugin/mdip.vim:91]: That was a long decision, I was in doubts. Finally, I think that it will be too hard to maintain windows support for me...

**Pull request closed without merging**